application/src/main/java/org/thingsboard/server/service/install/update/DefaultDataUpdateService.java

```diff
@@ -182,13 +182,22 @@ public void updateData(String fromVersion) throws Exception {
                 break;
             case "3.4.1":
                 boolean skipAuditLogsMigration = getEnv("TB_SKIP_AUDIT_LOGS_MIGRATION", false);
-                if (!skipAuditLogsMigration) {
+                boolean skipEdgeEventsMigration = getEnv("TB_SKIP_EDGE_EVENTS_MIGRATION", false);
+                if (!skipAuditLogsMigration || !skipEdgeEventsMigration) {
```

Comment:
please make it in sync with PE version - just print "Updating data from version 3.4.1 to 3.4.2 .." in front of case